gensim/similarities/docsim.py

```diff
@@ -92,22 +99,32 @@ def __init__(self, fname, index):
         self.index = self.get_index()
 
     def fullname(self):
+        """Return fullname of the corpus.
+        """
```

Comment:
if docstring < 120 characters, please write it in one line: `"""Return fullname of the corpus."""`

Reply:
also, try to avoid 'return' word, use 'get' for example (here and everywhere).

Reply:
besides, missing `Returns` section (here and everywhere).

Reply:
Okay, will do the fixes